### PR TITLE
Drop rackspace packages

### PR DESCRIPTION
**Pull request by tbrisker**
Core PR: https://github.com/theforeman/foreman/pull/7612

### CODE REVIEW

**Comment by tbrisker** [comment]
updated with obsoletes

File: packages/foreman/tfm/tfm.spec [hunk 1]
```diff
@@ -309,6 +310,9 @@ selinuxenabled && load_policy || :
 %{_root_sysconfdir}/rpm/macros.%{scl_name}-scldevel
 
 %changelog
+* Tue Apr 28 2020 Tomer Brisker <tbrisker@gmail.com>
```
Comment (ekohl) [inline packages/foreman/tfm/tfm.spec:313]:
```suggestion
* Tue Apr 28 2020 Tomer Brisker <tbrisker@gmail.com> - 6.0-5
```

Comment (tbrisker) [reply packages/foreman/tfm/tfm.spec]: hmm, odd, i used obal to do the rev bump, strange it didn't add the version here.

**Comment by tbrisker** [comment]
builds are failing due to rails 6 upgrade, package deps needs updating

**Comment by tbrisker** [comment]
[test rpm]

**Comment by tbrisker** [comment]
[test rpm] 
* note to self: wait for source release to finish first....

**Comment by zjhuntin** [comment]
This passed, older builds are just making it look red.

**Pull request merged**